Azure-Samples/chat-with-your-data-solution-accelerator
# Teams Extension enhancement

## Purpose

* Provide the ability to view the citation within Teams using an Adaptative Card.
* Update Teams Extension backend endpoint 
* Remove  Backend Function for Teams Extension.
* Remove  Backend Docker container for Teams Extension.
* Update Azure OpenAi Api Version to `2023-12-01-preview`

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  After deploying the code and installing the app within Teams, open the app and ask a question. Assuming a valid response, verify the citation displays within the Adaptive Card.

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

## Other Information
